jgrapht-core/src/main/java/org/jgrapht/alg/chordal/BucketList.java

```diff
@@ -0,0 +1,281 @@
+/*
+ * (C) Copyright 2018, by Timofey Chudakov and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.chordal;
```

Comment:
It might be a little to early to introduce a dedicated chordal package. For now I would include it in the cycle package (after all, a chordal graph is a graph in which all cycles of four or more vertices have a chord). Introducing a new (chordal) package can be done later when it becomes necessary.

Reply:
makes sense